src/lib/config-i18n/src/en.ts

```diff
@@ -475,6 +475,10 @@ translate(c.SubnetDefinitionConfig, {
       description:
         'Define the subnet in the Accelerator to reserve the address space, but do not create it at this time.  Used to define a 3 AZ config, but deploy a 2 AZ config.',
     },
+    'outpost-arn': {
+      title: '',
```

Comment:
Can we make sure we always provide a proper and detailed description for all new config file fields.  It should contain a valid example or a list of allowable values when controlled (like type) below.

Reply:
Added Comments